src/components/DefineDissolution/DestroyCertificate.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div>
+    <div v-if="isSummary" class="destroy-certificate-summary">
```

Comment:
If this can be "one of many instances" then a classname is appropriate. In this case, however, I think an id is more appropriate.